src/imodmodel/models.py

```diff
@@ -264,4 +265,10 @@ def from_file(cls, filename: os.PathLike):
         from .parsers import parse_model
         with open(filename, 'rb') as file:
             return parse_model(file)
-
+    
+    def to_file(self, filename: os.PathLike):
+        """Write an IMOD model to disk."""
+        from .writers import write_model
+        self.header.objsize = len(self.objects)
```

Comment:
Not quite sure if this should be here. I think ideally this should get changed whenever the `object`  list is modified, but I am unsure from top of my head if this is doable with getter/setters or other mechanism.

Reply:
I think this would be a place to use a [model_validator](https://docs.pydantic.dev/latest/concepts/validators/#model-validators) but this is fine for now 🙂 I'm not puhing for the model validator as it wouldn't ensure it was correct when writing, only on model construction...